<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>506</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lisa Backup - woof!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Sources</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListView" name="sourcesListView"/>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeSourceButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>196</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSourceDetails">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>451</width>
           <height>300</height>
          </size>
         </property>
         <property name="title">
          <string>Details</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>10</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>66</height>
             </size>
            </property>
            <property name="title">
             <string>Method</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radioButtonAll">
               <property name="text">
                <string>All</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonSelective">
               <property name="text">
                <string>Selective</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="minimumSize">
             <size>
              <width>409</width>
              <height>187</height>
             </size>
            </property>
            <property name="title">
             <string>Criteria</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <widget class="QComboBox" name="comboBoxDepth">
             <property name="geometry">
              <rect>
               <x>11</x>
               <y>50</y>
               <width>117</width>
               <height>23</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>Only root</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Direct children</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Recursive</string>
              </property>
             </item>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>11</x>
               <y>30</y>
               <width>37</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Depth</string>
             </property>
             <property name="buddy">
              <cstring>comboBoxDepth</cstring>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>12</x>
               <y>100</y>
               <width>56</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Predicate</string>
             </property>
             <property name="buddy">
              <cstring>comboBoxPredicate</cstring>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBoxPredicate">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>120</y>
               <width>158</width>
               <height>23</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>Contains file</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Name matches regex</string>
              </property>
             </item>
            </widget>
            <widget class="QStackedWidget" name="stackedWidgetPredicate">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>80</y>
               <width>241</width>
               <height>71</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>161</width>
                 <height>23</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>57</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="text">
                <string>Filename</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_4</cstring>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_2">
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>221</width>
                 <height>23</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_5">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>57</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="text">
                <string>Regex</string>
               </property>
              </widget>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxDestination">
         <property name="title">
          <string>Destination</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Root path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Triggers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="toolTip">
                 <string>Systemd identifier of the backup storage medium</string>
                </property>
                <property name="text">
                 <string>Backup device</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditSystemdUnit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QPushButton" name="pushButtonSelectDevice">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>122</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Install trigger</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
